.gitignore

```diff
@@ -9,3 +9,4 @@
 node_modules/
 vendor/
 .terraform/
+*.tfvars
```

Comment:
Technically, *.tfvars is not a "dependency directory". You could also put it under the "environment files" section, though that is not technically accurate either.
```suggestion

# Terraform variable files often contain secrets
*.tfvars
```